### PR TITLE
Fixed mixing project and directory names. 

### PR DESCRIPTION
**Pull request by mhotan**
Having to think of sub project naming is code smell.

### CODE REVIEW

File: balboa-client-dispatcher/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" % "balboa-dispatcher-client" %% "[0.16, )",
```
Comment (coyotebush) [inline balboa-client-dispatcher/README.md]: Will conflict with #23 but that's okay. Why different version specs in the two READMEs?

Comment (mhotan) [inline balboa-client-dispatcher/README.md]: No good reason but we should settle on a convention.  Must consult SBT docs.

Comment (coyotebush) [inline balboa-client-dispatcher/README.md]: And then let's pick one PR or the other that will change the dependency lines to the form
`"com.socrata" %% "balboa-client-dispatcher" % "0.16.+",`
(updated names, versions, and syntax)

Comment (mhotan) [inline balboa-client-dispatcher/README.md]: Lets update #23 to update all the README to use the + notation for those who are not intuned with mathematical notation.

**Comment by mhotan** [comment]
@housepage Can you give this the final blessing so we can begin to clean up balboa (Incrementally).

**Comment by housepage** [comment]
:two_men_holding_hands:

**Pull request merged**